src/main/resources/bootstrap.yaml

```diff
@@ -7,6 +7,7 @@ spring:
         case-documents-api-url: IA_CASE_DOCUMENTS_API_URL
         case-notifications-api-url: IA_CASE_NOTIFICATIONS_API_URL
         case-payments-api-url: IA_CASE_PAYMENTS_API_URL
+        home-office-integration-api-url: IA_HOME_OFFICE_INTEGRATION_API_URL
```

Comment:
This value is not used anywhere because you set the value in the values.yaml directly. Therefore, we can clean it up.

Reply:
Removed entry

Reply:
There was another review to not bump the version in Charts. I clicked something and it is removed. I have implemented that change too